.github/workflows/ci.yml

```diff
@@ -65,3 +65,15 @@ jobs:
           -Psigning.keyId="${{ secrets.SIGNING_KEY_ID }}"
           -Psigning.password="${{ secrets.SIGNING_PASSWORD }}"
           -Psigning.secretKeyRingFile="$HOME/secring.gpg"
+
+      - name: PublishDocumentation
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          ./gradlew dokkaHtml
+          git config user.email "ci-build@juul.com"
+          git config user.name "ci-build"
+          git checkout --orphan gh-pages
```

Comment:
We shouldn't need `--orphan` (that was just used to create the branch), right?

Reply:
Ah, I misunderstood what orphan was doing then. Will remove.